source/Outfit.h

```diff
@@ -105,6 +107,8 @@ class Outfit : public Weapon {
 	std::string displayName;
 	std::string pluralName;
 	std::string category;
+	std::string series;
+	int index;
```

Comment:
It's not obvious from their names what these are used for, so I think a comment is appropriate.
```suggestion
	// The series that this outfit is a part of and its index within that series.
	// Used for sorting within shops.
	std::string series;
	int index;
```